tests/worker.test.ts

```diff
@@ -62,10 +72,12 @@ function setupPiscina(workers: number, code: string, tasksPerWorker: number) {
 }
 
 test('piscina worker test', async () => {
-    const coreCount = os.cpus().length
+    // Uncomment this to become a 10x developer and make the test run just as fast!
+    const isDevRun = false
```

Comment:
What is this for?

Reply:
I added an extra line to describe it. Basically when working on tests and running them many times in a row, it made sense to run a "light" version of the test.

```
    // Uncomment this to become a 10x developer and make the test run just as fast!
    // Reduces events by 10x and limits threads to max 8
    const isLightDevRun = false
```